sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1354,6 +1506,12 @@ Exceptions:
 
 Returns: A new executable graph object which can be submitted to a queue.
 
+Exceptions:
+
+* Throws with error code `invalid` if the graph contains
+<<graph-memory-allocations, graph-owned memory allocations>> and any instance of
+an executable graph created from this modifiable graph is still alive.
```

Comment:
Add this exception to the previous list of "Exceptions" rather than defining a second "Exceptions" section.

Reply:
Fixed, thanks!